pc_ble_driver_py/ble_driver.py

```diff
@@ -1467,6 +1467,26 @@ def to_c(self):
             char_md.p_sccd_md = self.sccd_md.to_c()
         return char_md
 
+class BLEGattsValue(object):
+    def __init__(self, value=None, offset=0):
+        if value == None:
+            value = [0]
```

Comment:
What is the reason for why you want the default value to be a list with 0 elements? 

Also, do you support the functionality for when p_value is set to NULL(https://infocenter.nordicsemi.com/topic/com.nordic.infocenter.s132.api.v5.0.0/structble__gatts__value__t.html)?

Reply:
One version of the underlying c-function can be found at
[ble_gatts_value_set](https://infocenter.nordicsemi.com/topic/com.nordic.infocenter.s132.api.v5.0.0/group___b_l_e___g_a_t_t_s___f_u_n_c_t_i_o_n_s.html#ga2760c51ea71853bd74e2e7e7117ef52a)